views/lexicon/tabs/buttons_labels.html.twig

```diff
@@ -7,7 +7,7 @@
 
     <h2 class="heading">{{ html.icon('info-sign', { 'class': 'sidebar-icon'}) }} Some button help</h2>
 
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In tellus dolor, condimentum efficitur arcu sed, feugiat pellentesque lacus. Pellentesque ac ipsum ac mauris cursus volutpat. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.</p>
+    <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In tellus dolor, condimentum efficitur arcu sed, feugiat pellentesque lacus. Pellentesque ac ipsum ac mauris cursus volutpat. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.</div>
```

Comment:
Why change from a `p`?

Reply:
SRs often have shortcuts for selecting/reading paragraphs.

Reply:
No idea... reverted